app/common/controllers/header.js

```diff
@@ -27,7 +25,6 @@ angular.module('RedhatAccess.header').controller('HeaderController', [
         $scope.init = function () {
             HeaderService.pageLoadFailure = false;
             HeaderService.showPartnerEscalationError = false;
-            CaseService.sfdcIsHealthy = COMMON_CONFIG.sfdcIsHealthy;
```

Comment:
@vrathee everything looks fine, just ensure CaseService.sfdcIsHealthy is not being used any where outside common module. I doubt it but please check. If it is , replace with COMMON_CONFIG.sfdcIsHealthy.